nms/app/components/insights/GatewayMetrics.tsx

```diff
@@ -39,7 +39,7 @@ export default function (props: {configs: Array<MetricGraphConfig>}) {
     return <LoadingFiller />;
   }
 
-  const gatewayNames = Object.keys(selectors);
+  const gatewayNames = Object.keys((selectors as any).gateways);
```

Comment:
Do you still get linter issues if you simply do
```suggestion
  const gatewayNames = Object.keys(selectors.gateways);
```
?